.pre-commit-config.yaml

```diff
@@ -53,6 +53,7 @@ repos:
     rev: 23.9.1
     hooks:
       - id: black
+        entry: black --line-length 79 --preview
```

Comment:
I think you should use this:
https://pre-commit.com/#pre-commit-configyaml---hooks